apps/web/src/components/ReportModal/ReportModal.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { useTranslation } from 'next-i18next';
+
+import { Modal, Button } from 'ui';
+
+type ReportModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  reportProjectClick: (error: string, message: string) => void;
+};
+
+function ReportModal({
+  isOpen,
+  onClose,
+  reportProjectClick,
+}: ReportModalProps) {
+  const [selected, setSelected] = useState(null);
+  const [message, setMessage] = useState('');
+  const [error, setError] = useState(false);
+
+  const { t } = useTranslation('report-modal');
+
+  useEffect(() => {
+    setSelected(false);
+    setMessage('');
+  }, [isOpen]);
+
+  const TAGS = [
+    {
+      name: 'spam',
+      value: 'Spam',
+    },
+    {
+      name: 'nudity',
+      value: 'Nudity',
+    },
+    {
+      name: 'scam',
+      value: 'Scam',
+    },
+    {
+      name: 'illegal',
+      value: 'Illegal',
+    },
+    {
+      name: 'violence',
+      value: 'Violence',
+    },
+    {
+      name: 'suicide-or-self-injury',
+      value: 'Suicide or self injury',
+    },
+    {
+      name: 'hate-speech',
+      value: 'Hate speech',
+    },
+    {
+      name: 'something-else',
+      value: 'Something else',
+    },
+  ];
```

Comment:
I think you only need one value here, the translation key, so this can be an array of strings

Reply:
i need the value so that i can use it in the api to report